package.json

```diff
@@ -54,29 +54,29 @@
     "@types/ncp": "^2.0.1",
     "@types/node": "^8.0.32",
     "@types/once": "^1.4.0",
-    "@types/pify": "0.0.28",
+    "@types/pify": "3.0.0",
```

Comment:
Any reason not to have `^`?

Reply:
Good catch, this was automatically done by Greenkeeper. My suspicion is that this is an artifact of two separate mechanisms:

* Because 0.x releases are experimental and typically see breaking changes, npm doesn't add a `^` for packages below 1.0.0 installed with `npm install`
* Greenkeeper doesn't add or change `^` or `~`, apparently

I'll go ahead and add it.